src/main/java/io/javalin/websocket/WsContext.kt

```diff
@@ -32,6 +32,7 @@ abstract class WsContext(val sessionId: String, @JvmField val session: Session)
     fun queryString(): String? = upgradeCtx.queryString()
     fun queryParamMap(): Map<String, List<String>> = upgradeCtx.queryParamMap()
     fun queryParams(key: String): List<String> = upgradeCtx.queryParams(key)
+    fun queryParam(key: String): String = upgradeCtx.queryParam(key)
```

Comment:
```suggestion
    fun queryParam(key: String): String? = upgradeCtx.queryParam(key)
```

Build is currently failing because of this. Log: 
```
/home/travis/build/tipsy/javalin/src/main/java/io/javalin/websocket/WsContext.kt:[35,43] Type mismatch: inferred type is String? but String was expected
```